.rubocop.yml

```diff
@@ -23,15 +23,22 @@ Layout/EmptyLinesAroundModuleBody:
 Layout/EmptyLinesAroundClassBody:
   Enabled: false
 
-Layout/IndentHash:
-  Enabled: false
+Layout/ExtraSpacing:
+  Enabled: true
+
+Layout/TrailingBlankLines:
+  Enabled: true
```

Comment:
Both of these are enabled by default. Check with https://rubocop.readthedocs.io before adding stuff.

Reply:
should we copy the defaults into this file so we have a single source of truth?

Reply:
I think that'd make it very hard to maintain, e.g. when they add, remove or change cops.

Reply:
hmm, yeah, just concerned about things changing under us - by falling back to their defaults I think we risk them changing (and us having to override that later on) - but that'll be rare hopefully!